shared/folders/index.js

```diff
@@ -1,10 +1,10 @@
 import React, {Component} from 'react'
```

Comment:
fixed https://github.com/keybase/client/pull/2816/commits/383f963116c037d2d169a611c44fc5e8481242d5
we aren't integrated yet, but yeah. we should talk about a strategy to get everyting passing flow